DSharpPlus.Lavalink/Entities/LavalinkPlayer.cs

```diff
@@ -0,0 +1,273 @@
+        public LavalinkChannelMixFilter? ChannelMix { get; internal set; }
+
+        /// <summary>
+        /// Lets you filter higher frequencies
+        /// </summary>
+        [JsonProperty("lowPass", NullValueHandling = NullValueHandling.Ignore)]
+        public LavalinkLowPassFilter? LowPass { get; internal set; }
+
+        /// <summary>
+        /// Returns a new instance of <see cref="LavalinkFilters"/> with default values
+        /// </summary>
+        /// <returns>Default values for <see cref="LavalinkFilters"/></returns>
+        public static LavalinkFilters DefaultValues() =>
+            new()
+            {
+                Volume = 1.0f,
+                Equalizers = Array.Empty<LavalinkBandAdjustment>(),
+                Karaoke = new LavalinkKaraokeFilter(),
+                Timescale = new LavalinkTimescaleFilter(),
+                Tremolo = new LavalinkTremoloFilter(),
+                Vibrato = new LavalinkVibratoFilter(),
+                Rotation = new LavalinkRotationFilter(),
+                Distortion = new LavalinkDistortionFilter(),
+                ChannelMix = new LavalinkChannelMixFilter(),
+                LowPass = new LavalinkLowPassFilter()
+            };
+
+        public static LavalinkFilters FromFilters(IEnumerable<ILavalinkFilter> filters, IEnumerable<LavalinkBandAdjustment> equalizers = null)
+        {
+            if (filters == null)
+                throw new ArgumentNullException(nameof(filters));
+
+            var lavalinkFilters = new LavalinkFilters();
+
+            if (equalizers != null)
+                lavalinkFilters.Equalizers = equalizers;
+            foreach (var filter in filters)
+            {
+                if (filter is LavalinkKaraokeFilter karaokeFilter) lavalinkFilters.Karaoke = karaokeFilter;
+                if (filter is LavalinkTimescaleFilter timescaleFilter) lavalinkFilters.Timescale = timescaleFilter;
+                if (filter is LavalinkTremoloFilter tremoloFilter) lavalinkFilters.Tremolo = tremoloFilter;
+                if (filter is LavalinkVibratoFilter vibratoFilter) lavalinkFilters.Vibrato = vibratoFilter;
+                if (filter is LavalinkRotationFilter rotationFilter) lavalinkFilters.Rotation = rotationFilter;
+                if (filter is LavalinkDistortionFilter distortionFilter) lavalinkFilters.Distortion = distortionFilter;
+                if (filter is LavalinkChannelMixFilter channelMixFilter) lavalinkFilters.ChannelMix = channelMixFilter;
+                if (filter is LavalinkLowPassFilter lowPassFilter) lavalinkFilters.LowPass = lowPassFilter;
+            }
+
+            return lavalinkFilters;
+        }
+    }
+
+    public class LavalinkPlayerUpdatePayload
+    {
+        /// <summary>
+        /// The encoded track base64 to play. null stops the current track
+        /// </summary>
+        [JsonProperty("encodedTrack", DefaultValueHandling = DefaultValueHandling.Ignore)]
+        [DefaultValue("")]
```

Comment:
Why do we have a `DefaultValue` attribute?

Reply:
Because if we send null to the server, playback will be stopped.

Reply:
Is setting `EncodedTrack = "";` not enough?